### DISCUSSION TITLE
Wrong explanation for RunOut: True it should be RunOut: False

### DISCUSSION THREAD

**Discussion by Jaz939** [boschresearch/pylife#76] [General]
Hi,
In the beginning, I would like to thank you for the great script.

When I reworked your demo script woehler_analyzer and data set fatigue-data-fractures, I noticed that in the library in script fatigue_data.py you have wrongly addressed the boolean for RunOut. You have stated that RunOut is True. When you do this the finite zone and infinite zone are wrong.  
The correct value for RunOut is False. Then the program is working correctly. 

Again thanks for the script.
Roman

**Comment by johannes-mueller** [comment]
Thanks for pointing this out. We will take care of it. #77

**Reply by johannes-mueller** [reply]
@Jaz939 please check [this change](https://github.com/boschresearch/pylife/pull/79/files). Is that what you meant or have you seen a similar error somewhere else?
